_docker/docker-compose.yml.erb

```diff
@@ -154,7 +154,23 @@ services:
       - DB_PASSWORD=searchisko
     links:
      - mysql
-  
+
+  #
+  # Environment actions
+  #
+  backup:
+    build: ./awestruct
```

Comment:
Primarily due to the way in which we're handling Gemfiles at the minute. control.rb is copying in Gemfile and Gemfile.lock from the project root to keep things "in-sync" when the awestruct image gets built. It just felt easier to adopt this for now.

What I'd like to do at a later date is pull all of these actions e.g. backup, restore, export out into their own Git repository and distribute them as an image. When we do that then we can:

- Have our own Gemfile and Gemfile.lock for just that project
- Distribute them on a Docker image that only has the bare-minimum. I actually like the idea of using https://github.com/docker-library/ruby/blob/5d04363db6f7ae316ef7056063f020557db828e1/2.1/onbuild/Dockerfile as it makes it very easy to build Docker images that are basically ruby distributions.

So again its a bit of a "definitely room for improvement" type of scenario, which I think we can look at post v1.